scripts/st2bootstrap-deb.sh

```diff
@@ -79,6 +79,10 @@ install_st2mistral() {
   /opt/stackstorm/mistral/bin/mistral-db-manage --config-file /etc/mistral/mistral.conf upgrade head
   # Register mistral actions
   /opt/stackstorm/mistral/bin/mistral-db-manage --config-file /etc/mistral/mistral.conf populate
+  
+  # Start Mistral
+  sudo service mistral-server start
+  sudo service mistral-api start
 }
```

Comment:
i think it's ok: 
1) this won't work either https://github.com/StackStorm/st2-packages/blob/DZ/start_mistral_deb/scripts/st2bootstrap-deb.sh#L104
2) not sure if st2ctl will work. 
3) it's not typical, and if it happens folks will figure this out.